Writing/Twelve principles of time management.md

```diff
@@ -0,0 +1,38 @@
+
+---
+tags: management, writing, time
+author: Tran Hoang Nam
+---
+
+**12 Principles Of Time Management**
+
+ - Determine what is urgent
+ - Don’t over commit
+ - Have a plan for your time
+ - Allow time for the unexpected
+ - Handle things once
+ - Create realistic deadlines
+ - Set goals for yourself and your time
+ - Develop routines
+ - Focus on one thing at a time
+ - Eliminate or minimize distractions
+ - Outsource tasks or delegate
+ - Leave time for fun and play
```

Comment:
I think this checklist is just too generic and can be found easily anywhere. I would suggest it'd be better if we could go into a bit more detail or at least have some examples.